Objects/typeobject.c

```diff
@@ -6958,8 +6990,14 @@ update_one_slot(PyTypeObject *type, slotdef *p)
         return p;
     }
     do {
-        descr = _PyType_Lookup(type, p->name_strobj);
+        descr = _PyType_LookupUncached(type, p->name_strobj, &error);
```

Comment:
Would be nice to add a comment explaining why the uncached lookup.

Reply:
Sure enough. Added.